metricbeat/tests/system/test_docker.py

```diff
@@ -12,24 +12,13 @@ def test_container_fields(self):
         """
         test container fields
         """
-
-        test_env = os.environ.get('DOCKER_COMPOSE_PROJECT_NAME')
```

Comment:
remove on purpose?

Reply:
Yes, locally I get: `DOCKER_COMPOSE_PROJECT_NAME=metricbeatsnapshot-noxpack7.0.0-alpha111d177efe455c1f207da557ad4e2f2f3f1981c53`. But docker-compose strips down some chars and it ends up in: `metricbeatsnapshotnoxpack700alpha111d177efe455c1f207da557ad4e2f2f3f1981c53`.

Of course, I could strip them down, but this test doesn't really care about what containers it is retrieving, as long as it gets some